zebra/router-id.c

```diff
@@ -253,6 +253,34 @@ DEFUN (no_router_id,
 	return CMD_SUCCESS;
 }
 
+DEFUN (show_router_id,
+       show_router_id_cmd,
+       "show router-id [vrf NAME]",
+       "Show the configured router-id\n"
+       VRF_CMD_HELP_STR)
+{
+        int idx_name = 3;
+
+        vrf_id_t vrf_id = VRF_DEFAULT;
+        struct zebra_vrf *zvrf;
+
+        if (argc > 2)
```

Comment:
Tabs

Reply:
@qlyoung 
This file is filled with tabs! want me to fix'em all? or should I keep those untouched/separate? usually I'd say  keep them separate but this is a small and isolated file.